### DISCUSSION TITLE
Lidar - how to activate ?

### DISCUSSION THREAD

**Discussion by ghost** [iNavFlight/inav#9158] [General]
Having installed Lidar Matek system:
How has the arrow which is shown at Lidar be placed? In General to the front of the system, or in same direction as FC is mounted?

Could Lidar be used without Mag? How to activate corretly?
Lidar to be calibrated with calibration method´.
How to activate then Lidar, that copter stayes stable in the air?

**Comment by ghost** [comment]
How do I get detailed informations to set Lidar correctly up?
I installed, but its not working (so copter is not stable at all)
What can I do? Where can I  get details about it?

**Reply by terryfritz99** [reply]
This document http://www.mateksys.com/?portfolio=3901-l0x#tab-id-2 from the Matek website explains how to  set up, calibrate and use their 3901-LOX module. Chief point is that if your FC direction arrow and the lidar module arrow are on the top and the same direction the “align_opflow” setting is CW0FLIP. I found that writing the Tuning parameters at the bottom of the document in CLI improved the terrain following performance. Over a high contrast surface I can travel at 70cm above the surface. You have to turn on surface mode before Altitude or Position Hold with a separate switch. Hope this helps.
